### PR TITLE
stumpish: init at 0.0.1

### PR DESCRIPTION
**Pull request by ebzzry**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/window-managers/stumpish/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "stumpwm";
+    repo = "stumpwm-contrib";
+    rev = "9f5f06652c480159ec57d1fd8751b16f02db06dc";
+    sha256 = "1dxzsnir3158p8y2128s08r9ca0ywr9mcznivmhn1lycw8mg4nfl";
+  };
+
+  phases = [ "buildPhase" "installPhase" "postFixup" ];
```
Comment (jtojnar) [inline pkgs/applications/window-managers/stumpish/default.nix]: Manually setting phases is frowned upon https://github.com/NixOS/nixpkgs/issues/28910

**Comment by ebzzry** [comment]
@jtojnar I updated `default.nix`. However, why is the file not being properly patched?

**Comment by jtojnar** [comment]
It does seem so.

**Comment by ebzzry** [comment]
@jtojnar Thanks!

**Comment by ebzzry** [comment]
@jtojnar Thanks!

**Pull request merged**